modules/UI/authentication/AuthHandler.js

```diff
@@ -3,11 +3,14 @@
 import Logger from 'jitsi-meet-logger';
 
 import { openConnection } from '../../../connection';
-import { externalAuthDialog } from '../../../react/features/authentication/actions';
+import { _openLoginDialog } from '../../../react/features/authentication/actions.native';
```

Comment:
Why are we importing things from native?

Reply:
Created actions.any.js for actions used on both web and mobile.